alacritty/src/config/mod.rs

```diff
@@ -125,6 +125,8 @@ pub fn load(options: &Options) -> UiConfig {
 
 /// Attempt to reload the configuration file.
 pub fn reload(config_path: &Path, options: &Options) -> Result<UiConfig> {
+    debug!("Reloading configuration file from: {:?}", config_path);
```

Comment:
```suggestion
    debug!("Reloading configuration file: {:?}", config_path);
```

Don't really care, but I think this is better with the colon.